src/odemis/gui/comp/viewport.py

```diff
@@ -407,6 +408,12 @@ def _checkMergeSliderDisplay(self):
                 ):
                     self.bottom_legend.set_stream_type(wx.LEFT, model.MD_AT_EM)
                     self.bottom_legend.set_stream_type(wx.RIGHT, model.MD_AT_SPECTRUM)
+                elif (
```

Comment:
Is ARStream is ever displayed on top of a SEM stream? Did you see the AR icon being displayed on the wrong side, or is it just from the code? If it's the later one, probably this commit is not needed.